src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DeleteIT.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Locale;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class DeleteIT extends SQLIntegTestCase {
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ACCOUNT);
+        loadIndex(Index.PHRASE);
+    }
+
+    @Test
+    public void deleteAllTest() throws IOException, InterruptedException {
+        String selectQuery = String.format(
+            Locale.ROOT,
+            "SELECT * FROM %s/account",
+            TestsConstants.TEST_INDEX_ACCOUNT
+        );
+        JSONObject response = executeRequest(makeRequest(selectQuery));
+        int totalHits = getTotalHits(response);
+
+        String deleteQuery = String.format(
+            Locale.ROOT,
+            "DELETE FROM %s/account",
+            TestsConstants.TEST_INDEX_ACCOUNT);
+        response = executeRequest(makeRequest(deleteQuery));
+        assertThat(response.getInt("deleted"), equalTo(totalHits));
+        Thread.sleep(2000);
```

Comment:
can you add a comment to explain why we need to do `sleep`

Reply:
Sure, will do.